includes/tax-functions.php

```diff
@@ -415,12 +415,17 @@ function edd_download_is_tax_exclusive( $download_id = 0 ) {
  * Gets the tax rate object from the database for a given country / region.
  * Used in `edd_get_tax_rate`, `edd_build_order`, `edd_add_manual_order`.
  *
- * @param array $args
+ * @param array $args {
+ *     Country and, optionally, region to get the tax rate for.
+ *
+ *     @type string $country Required - country to check.
+ *     @type string $region  Optional - check a specific region within the country.
+ * }
  * @return object|boolean
  *
  * @since 3.0
  */
-function edd_get_tax_rate_object( $args ) {
+function edd_get_tax_rate_for_location( $args ) {
```

Comment:
I guess another question here is should the parameters be updated to be more consistent with the `edd_get_tax_rate` function, which has these as parameters:

- `$country = ''`
- `$region = '`
- `$fallback = true` (would not need)

So pass each as its own item instead of array keys? Or does it matter?

Reply:
1. Suggest to keep this as an `$args` array so keys can be added later without changing the function signature.
1. Maybe should set `$args = array()` in the function parameters, and then use `$r = wp_parse_args( $args, array( whatever ) )` internally. That way the default keys and values are already taken care of, in the event something weird is passed in or the defaults need to change later on?
1. `_for_location()` is better than the original `_object()`, but it kinda hints to me that it's for a literal location, like an address. (I guess if it were for an address, it would be `_by_address()`, but that function doesn't exist either; it's just part of `edd_get_tax_rate()`.) I can't really think of anything better though.

Reply:
I think the tax rate in 3.0 requires a location now--if the storefront has a default/global rate set, there are warnings to remove it. So an address (that is, country at minimum) is required to get the tax rate object.

Reply:
I think using `_by_` is more consistent with the existing `edd_get_x_by_y()` patterns we use elsewhere. `edd_get_tax_rate_by_location()` makes sense to me.